.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,95 @@
+name: Bug Report
+description: File a bug report to help improve qBittorrent user experience.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        #### ADVISORY
+        "We do not support any versions older than the current release series"
+
+        "We do not support any 3rd party/forked versions e.g. `portableapps`/`Enhanced Edition`etc."
+
+        "Please post all details in **English**."
+
+  - type: checkboxes
+    attributes:
+      label: Mandatory prerequisites
+      description: To rule out invalid issues, confirm and check each one of the required checkboxes.
+      options:
+        - label: I have read the issue reporting section in the **[contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md)**, so I know how to submit a good bug report with the required information.
+          required: true
+        - label: I have verified that the issue is not fixed and is reproducible in the **[latest official qBittorrent version](https://www.qbittorrent.org/download.php).**
+          required: true
+        - label: I have checked the **[frequent/common issues list](https://github.com/qbittorrent/qBittorrent/projects/2)** and performed a **[search of the issue tracker (including closed ones)](https://github.com/qbittorrent/qBittorrent/issues)** to avoid posting a duplicate.
+          required: true
+        - label: This is not a support request or question, both of which are better suited for either the **[discussions section](https://github.com/qbittorrent/qBittorrent/discussions)**, **[forum](https://qbforums.shiki.hu/)**, or **[subreddit](https://www.reddit.com/r/qBittorrent/)**. The **[wiki](https://github.com/qbittorrent/qBittorrent/wiki)** did not contain a suitable solution either.
+          required: true
+        - label: I understand that it is the responsibility of the user to **redact any sensitive information** in logs/config files.
+          required: true
+        - label: I understand that if relevant to issue/when asked, I must provide the **qBittorrent.ini**, **qBittorrent.log** and **watched_folders.json** file(s) (if using "Watched Folders" feature). See **[Where does qBittorrent save its settings?](https://github.com/qbittorrent/qBittorrent/wiki/Frequently-Asked-Questions#Where_does_qBittorrent_save_its_settings)**
```

Comment:
Maybe=
qBittorrent settings (.ini/.conf/.idk)

Reply:
>Why split the above checkbox from this one? Seems superfluous.

Will integrate.

>I would change "qBittorrent.ini" to "qBittorrent settings|config|preferences file" (pick one) because it's not ".ini" in linux, it's ".conf" (maybe mac too idk).

I will choose **qBittorrent preferences** as that's what's used if you follow [Where does qBittorrent save its settings?](https://github.com/qbittorrent/qBittorrent/wiki/Frequently-Asked-Questions#Where_does_qBittorrent_save_its_settings)

Will also change description under Log(s) & settings file(s) to reflect same.

Reply:
Maybe just me, if it is really necessary, I would use code blocks instead of bold text for "`qBittorrent preferences file`/`qBittorrent.log` and `watched_folders.json`"

Reply:
It's not 100% necessary but It's a clearer visual (to me anyway) of what files are required, code blocks work too.

I can omit any kind of formatting on them if you prefer?

Reply:
>I can omit any kind of formatting/highlighting on them if you prefer?

That will work for me too.

Reply:
all issues addressed I believe.